thrift/linkattacher/CMakeLists.txt

```diff
@@ -0,0 +1,29 @@
+# Copyright (C) 2016 Istituto Italiano di Tecnologia iCub Facility & RBCS Department
+# Authors: Yeshasvi Tirupachuri
+# CopyPolicy: Released under the terms of the LGPLv2.1 or later, see LGPL.TXT
+
+cmake_minimum_required(VERSION 2.8.11)
+
+set(LIBRARY_TARGET_NAME gazebo_yarp_rpc_linkattacher)
+
+yarp_idl_to_dir(linkattacher_rpc.thrift ${CMAKE_CURRENT_SOURCE_DIR}/autogenerated LARPC_SRC LARPC_HEADERS LARPC_INCLUDES)
+
+include_directories(${LARPC_INCLUDES} ${YARP_INCLUDE_DIRS})
```

Comment:
`YARP_INCLUDE_DIRS` is deprecated. Other parts of this project need to be updated. At least for new code let's try to avoid deprecated functionalities. Linking yarp libraries should be enough because targets import their include directories.

Reply:
Furthermore, including `${LARPC_INCLUDES}` shouldn't be necessary since at line 19 you already add them to the target

Reply:
Done. I also removed `YARP_INCLUDE_DIRS` from plugin `CMakeLists.txt`